packages/web/src/services/entity-manager.ts

```diff
@@ -22,6 +22,7 @@ export const makeEntityManagerInstance = (
     contractAddress: servicesConfig.entityManagerContractAddress,
     web3ProviderUrl: servicesConfig.web3ProviderUrl,
     identityServiceUrl: servicesConfig.identityServiceUrl,
+    useDiscoveryRelay: true,
```

Comment:
Related to this change?

Reply:
yeah - will chat more offline. need some solution here though because this is definitely broken... at least for local dev!